orca-remote-stage/src/main/kotlin/com/netflix/spinnaker/orca/remote/pipeline/RemoteStage.kt

```diff
@@ -0,0 +1,37 @@
+package com.netflix.spinnaker.orca.remote.pipeline
+
+import com.netflix.spinnaker.kork.annotations.Beta
+import com.netflix.spinnaker.orca.api.pipeline.graph.StageDefinitionBuilder
+import com.netflix.spinnaker.orca.api.pipeline.graph.StageDefinitionBuilder.Aliases
+import com.netflix.spinnaker.orca.api.pipeline.graph.TaskNode
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution
+import com.netflix.spinnaker.orca.remote.model.RemoteStageExtensionConfig
+import com.netflix.spinnaker.orca.remote.service.RemoteStageExtensionService
+import com.netflix.spinnaker.orca.remote.tasks.MonitorRemoteStageTask
+import com.netflix.spinnaker.orca.remote.tasks.StartRemoteStageTask
+import org.springframework.stereotype.Component
+
+@Beta
+@Component
+@Aliases("remoteStage")
```

Comment:
Prior convention is that we do not include `Stage` in stage types, as that's redundant. I feel like we can get rid of this annotation entirely, as it'll already get the `remote` type from [`StageDefinitionBuilder.getType`](https://github.com/spinnaker/orca/blob/master/orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/graph/StageDefinitionBuilder.java#L90)